src/Action/Hello.action.ts

```diff
@@ -0,0 +1,20 @@
+import lambdaWrapper, {
+  RequestService,
+  ResponseModel,
+} from '@comicrelief/lambda-wrapper';
+
+/**
+ * An example handler
+ *
+ * @param {DependencyInjection} di
+ * @returns {Promise<ResponseModel>}
+ */
+export default lambdaWrapper.wrap(async (di) => {
+  const request = di.get(RequestService);
+
+  return ResponseModel.generate(
+    {},
+    200,
+    `hello ${request.get('name', 'nobody')}`,
```

Comment:
You've changed some functionality here, returning `hello nobody` instead of `Hello`. I'd prefer not to change anything beyond what is needed for the TypeScript migration. Doesn't matter so much here but is good practice for when we migrate other projects.

Reply:
Have modified this behaviour to reflect how it currently works.